sdk/tables/azure-data-tables/azure/data/tables/_authentication.py

```diff
@@ -16,7 +15,7 @@
 from azure.core.exceptions import ClientAuthenticationError
 from azure.core.pipeline.policies import SansIOHTTPPolicy
 
-from azure.data.tables._shared._constants import (
+from azure.data.tables._constants import (
```

Comment:
nit: this can also be a relative import

Reply:
nit: I can't comment on the line I'm referring to (I'm trying out the new vscode github extension), but at the bottom of this file there's a big chunk of comemnted out code that can just go